python/graphstorm/model/rgcn_encoder.py

```diff
@@ -272,33 +264,35 @@ def _apply(ntype, h):
 
 
 class RelationalGCNEncoder(GraphConvEncoder, GSgnnGNNEncoderInterface):
-    r""" Relational graph conv encoder.
+    """ Relational graph conv encoder.
 
-    The RelationalGCNEncoder employs several RelGraphConvLayer as its encoding mechanism.
-    The RelationalGCNEncoder should be designated as the model's encoder within Graphstorm.
+    The ``RelationalGCNEncoder`` employs several ``RelGraphConvLayer`` as its encoding
+    mechanism. The ``RelationalGCNEncoder`` should be designated as the model's encoder
+    within Graphstorm.
 
     Parameters
     ----------
-    g : DistGraph
-        The distributed graph object.
-    h_dim : int
-        Hidden dimension
-    out_dim : int
-        Output dimension
+    g: DistGraph
+        The distributed distributed graph.
+    h_dim: int
+        Hidden dimension.
+    out_dim: int
+        Output dimension.
     num_bases: int
-        Number of bases.
-    num_hidden_layers : int
-        Number of hidden layers. Total GNN layers is equal to num_hidden_layers + 1. Default 1
-    dropout : float
-        Dropout. Default 0.
-    use_self_loop : bool
-        Whether to add selfloop. Default True
-    last_layer_act : torch.function
-        Activation for the last layer. Default None
+        Number of bases. If is None, use number of relation types. Default: None.
+    num_hidden_layers: int
+        Number of hidden layers. Total GNN layers is equal to ``num_hidden_layers + 1``.
+        Default: 1.
+    dropout: float
+        Dropout rate. Default 0.
+    use_self_loop: bool
+        Whether to add selfloop. Default: True.
```

Comment:
In RelGraphConvLayer, it is false. Shall we be consistent and set it to True?

Reply:
Agree. Changed to `True` in `RelGraphConvLayer`.